src/diy/imu-comparison.md

```diff
@@ -34,60 +42,120 @@ They are reliable and stable chips, but at the time of writing, they remain diff
 
 |Reset time |Cost |Availability|Build quality|
 |:---------:|:---:|:----------:|:-----------:|
-|30 - 45 min|~$13 |Sufficient  |Excellent    |
+|45-60 min|~$10 |Sufficient  |Excellent    |
```

Comment:
```suggestion
|30-45 min|~$11 |Sufficient  |Excellent    |
```

Idk where you got BNO085 for $10. The cheapest is $11 on our site. Also the drift time is very subjective and there are a lot of arguments, so I wouldn't recommend increasing it for BNO just yet. We really need a well-defined measurement system for drift time...